rules.md

```diff
@@ -238,6 +238,12 @@ Griefing is the intent of one player wanting to cause grief or annoyance to othe
 - Shooting up medbay at round start.
 - Deliberately OBing the marine frontline.
 
+## Permanent Removal
+
+- Permanently removing a player from the round by using brute force, or preventing them from being treated, is considered as griefing, even if they belong to a hostile faction. 
```

Comment:
```suggestion
- Permanently removing a player from the round by using brute force, or preventing them from being treated, is considered as griefing, even if they belong to a hostile faction. 
- It is not considered griefing to ignore dead hostiles, only to take action to prevent them being treated.
```